core/generator.rb

```diff
@@ -100,7 +100,7 @@ def Generator.getVmDef(cookbook_path, name, host, boxurl, ssh_pty, vm_mem, templ
       templatedef = ''
     end
     # ssh.pty option
-    ssh_pty ? ssh_pty_option = "\tconfig.ssh.pty = true" : ssh_pty_option = ''
+    if ssh_pty == "true"; ssh_pty_option = "\tconfig.ssh.pty = true"; else ssh_pty_option = ''; end
```

Comment:
should it be in this way:
if ssh_pty == "true"  - Vagrantfile "\tconfig.ssh.pty = true"
if ssh_pty == "false" - Vagrantfile  "\tconfig.ssh.pty = false"

if there is no ssh_pty in the .json - nothing is added to Vagrant file 
?